challenges/classes.js

```diff
@@ -1,7 +1,27 @@
 // 1. Copy and paste your prototype in here and refactor into class syntax.
 
+class CuboidMaker{
+constructor (length, width, height) {
+  this.length = length;
+  this.width = width;
+  this.height = height;
+}
+
+    findVolume () {
+  return this.length * this.width * this.height;
+    }
+    
+    surfaceArea (){
+  return 2 * (this.length * this.width + this.length * this.height + this.width * this.height);
+}
+
+}
+
+let cuboid = new CuboidMaker(4, 5, 5); 
+
+
 // Test your volume and surfaceArea methods by uncommenting the logs below:
-// console.log(cuboid.volume()); // 100
-// console.log(cuboid.surfaceArea()); // 130
+console.log(cuboid.volume()); // 100
```

Comment:
careful the way you console.logged this...  needs to match your naming convention from above.